dd-sdk-android/src/test/kotlin/com/datadog/android/utils/forge/JsonPrimitiveForgeryFactory.kt

```diff
@@ -16,7 +16,8 @@ class JsonPrimitiveForgeryFactory : ForgeryFactory<JsonPrimitive> {
         return forge.anElementFrom(
             JsonPrimitive(forge.aBool()),
             JsonPrimitive(forge.anInt()),
-            JsonPrimitive(forge.aFloat()),
+            // TODO put it back once proper JSON assertions are ready
```

Comment:
Can you add the JIRA ticket number in your TODO please, e.g.:
```kotlin
    // TODO RUMM-1234 put it back once proper JSON assertions are ready
```

Reply:
done